### DISCUSSION TITLE
Project X入群 问题答案

### DISCUSSION THREAD

**Discussion by ghost** [XTLS/Xray-core#3562] [General]
> 如果在 Minecraft 1.20 使用用于生成NIST P-256所使用曲线E的种子，离出生点最近的要塞坐标是

## 建议换个问题 如
* Minecraft的开源替代品是 
答案 [Cubyz](https://github.com/PixelGuys/Cubyz) [Minetest](http://minetest.net/)

**Comment by liuchengts** [comment]
这就他妈的很扯淡了

**Comment by nekohalawrence** [comment]
逆天题目，真的不懂。🙃

**Reply by ghost** [reply]
> 逆天题目，真的不懂。🙃

@Fangliding

**Comment by creazy412** [comment]
已排雷3个：
-794,-1871
777,-1880
-1278,2126

**Reply by liuchengts** [reply]
不用排雷了，既然人家不欢迎，我就不进了，这种态度xray各自安好吧，我宁愿继续v2ray

**Comment by Kc2353** [comment]
~~感谢历年真题~~
~~为什么你们能抽到这么简单的~~
先问ai，得知这条曲线叫‘secp256r1’
之后搜索‘Minecraft地图查询’，结果是seedmap-chunkbase，seed,看起来就是上面说的种子，填入
版本选java1.20（默认是java最新版，那肯定选java1.20 ~~，其实Java版要塞（stronghold）位置的算法最近几个版本没变~~ ）
有三个比较近的 ~~（对于有所了解的人这是废话，必然有三个比较近的围成一圈）~~ ，拿计算器算一下，得出答案
<img width="2031" height="1293" alt="image" src="https://github.com/user-attachments/assets/b84b0901-47fc-4ab1-b625-86bddf7ea02d" />
